package.json

```diff
@@ -26,16 +26,16 @@
     "node": ">=4"
   },
   "dependencies": {
-    "follow-redirects": "0.0.7",
-    "https-proxy-agent": "^1.0.0",
-    "mime": "^1.4.0",
+    "follow-redirects": "1.2.5",
+    "https-proxy-agent": "^2.1.0",
+    "mime": "^2.0.3",
```

Comment:
I reviewed all changes and they shouldn't affect us. `https-proxy-agent` dropped Node v4 support, too, so we should do the same. Besides doing a breaking version bump just in case will help in case we have overseen a breaking change that affects node-github. I don`t trust our test suite yet